### ISSUE TITLE
AdminCLI "vmdkops_admin.py ls" failed when datastore name includes white space

### ISSUE THREAD

**Issue by lipingxue** [vmware/docker-volume-vsphere#892]
Steps to reproduce:
1. create several volumes in datastore (for example "tmp-(datastore1)"), then run vmdkops_admin.ls
```
[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Volume  Datastore         Created By VM  Created                   Attached To VM  Policy  Capacity  Used     Disk Format  Filesystem Type  Access      Attach As               
------  ----------------  -------------  ------------------------  --------------  ------  --------  -------  -----------  ---------------  ----------  ----------------------  
vol1    tmp-(datastore1)  photon5        Mon Sep  5 01:47:03 2016  detached        N/A     100.00MB  13.00MB  thin         ext4             read-write  independent_persistent  
vol2    tmp-(datastore1)  photon5        Mon Sep  5 01:48:43 2016  detached        N/A     100.00MB  13.00MB  thin         ext4             read-write  independent_persistent  
vol3    tmp-(datastore1)  photon5        Mon Sep  5 04:08:01 2016  detached        N/A     100.00MB  13.00MB  thin         ext4             read-write  independent_persistent  

```

2. rename the datastore name by adding extra whitespace in the datastore name  ("tmp(datastore1)" -> "tmp (datastore1)")
run the same command again, get the following error

```
[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Invalid destination specification:
vmkfstools: (datastore1)/dockvols/077d0f05-f262-4720-91b2-9d55a23870ff/vol1.vmdk: No name specification expected.
Failed to stat /vmfs/volumes/tmp- (datastore1)/dockvols/077d0f05-f262-4720-91b2-9d55a23870ff/vol1.vmdk. VMDK corrupted. Please remove and then retry

```

**Comment by shuklanirdesh82** [comment]
resolving duplicate of #816 (#816 should care of the reported issue)

**Comment by shuklanirdesh82** [comment]
reopening this one as per offline discussion with @lipingxue

**Comment by govint** [comment]
May be fixed by changes posted for #816

**Comment by govint** [comment]
Closing, as #911 is changing the same function thats causing the behavior to use KV API vs. command line vmkfstools util.

Currently its not possible to create a volume on a datastore with spaces in it. I'm raising a separate PR to handle that although why such names must be supported or even created is unclear. At least for Photon its not needed and mostly for the occasional  cases in ESX perhaps.
